### PR TITLE
Repaired seed file and changed rate per day

### PR DESCRIPTION
**Pull request by akilchomoko**
Correct the duplicate line and the rate per day calculation.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -60,6 +60,7 @@
     t.string "address2"
     t.string "postcode"
     t.string "phonenumber"
+    t.integer "imdb_id"
```
Comment (gtm19) [inline db/schema.rb]: @akilchomoko why does a user have an IMDB id?

**Review by gtm19 — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 1 above)
Comment (akilchomoko) [reply db/schema.rb]: It shouldn't. There should have been a migration that removed it from an earlier mistake.

**Comment by akilchomoko** [comment]
Corrected the user_id table. Removed the incorrect imdb_id column. I thought I had done that migration earlier and it was all sorted but may have lost that change in the git learning process. Should be sorted now.

**Review by gtm19 — APPROVED** [review]

**Comment by WaynePatrickDalton** [comment]
I just merged this to get us up-to-date.

**Pull request merged**